Clicker/Quarks/DraftQuark.swift

```diff
@@ -106,7 +106,6 @@ struct UpdateDraft: ClickerQuark {
     }
     var parameters: Parameters {
         return [
-            "id": id,
```

Comment:
why are we removing this?

Reply:
Yuna told me that we don't need to specify the id in that parameters variable, that the request will still work without it.